src/app/i18n.js

```diff
@@ -0,0 +1,49 @@
+import i18n from "i18next";
```

Comment:
Please could you investigate why we get this on server side:
![image](https://github.com/user-attachments/assets/6f7ab3ff-9fe8-4450-a087-289fe071f870)

Why does it query for translation.json?